allenact_plugins/ithor_plugin/ithor_environment.py

```diff
@@ -38,11 +38,16 @@ def __init__(
         fov: float = FOV,
         player_screen_width: int = 300,
         player_screen_height: int = 300,
+        gridSize: float = 0.15,
```

Comment:
Thanks, I updated the default values as you suggested. Where can I find the default value for visibilityDistance?

Reply:
I think it's actually 1.5:

https://ai2thor.allenai.org/ithor/documentation/initialization/#initialization-visibilitydistance

Other than the arguments that had already been set by Luca, I'd try to keep everything else matching the defaults in the link to keep all other experiment configs working as expected.

Reply:
I have reset all the default values in the environment init from the link. For the visibility_distance there was a constant defined in this file https://github.com/allenai/allenact/blob/main/allenact_plugins/ithor_plugin/ithor_constants.py which was equal to 1.25 so I am using that Constant instead otherwise all the values are set to default from the link.